Majoolr/ethereum-libraries
# Adds coverage report tool

## What's being added with this PR:

- Using https://github.com/sc-forks/solidity-coverage to generate the full report. 
- Using https://codecov.io as report host and analytics platform due to the lack of features on coveralls.
- Added a badge on the root directory to indicate the % of coverage of the overall libraries.

## How to test
- Enter a directory, for example, `WalletLib/truffle`.
- `npm install && npm install -g codecov` to cover any new lib.
- `npm run test:coverage` this commad will run the tests and generate the report after this.
- the report is placed on `coverage` folder.
- run `codecov`
- check https://codecov.io/gh/Majoolr/ethereum-libraries
